ajax/libs/paver/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "paver",
+  "filename": "js/jquery.paver.min.js",
+  "version": "1.3.3",
+  "description": "A minimal panorama/image viewer replicating the effect seen in Facebook Pages app.",
+  "homepage": "https://terrymun.github.io/paver/",
+  "author": {
+    "name": "Terry Mun",
+    "email": "hello@terrymun.com",
+    "homagepage": "http://terrymun.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/terrymun/paver/"
+  },
+  "keywords": [
+    "paver",
+    "panorama",
+    "photo"
+  ],
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/terrymun/paver.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
+        "files": [
+          "**/*.js"
```

Comment:
it's also a little bit not safe or no sense to use `**/` under a non-dist folder